yazi-boot/src/actions/debug.rs

```diff
@@ -79,17 +89,26 @@ impl Actions {
 			"    file             : {}",
```

Comment:
How can I align this `file` if there's a need to add in more deps?

Reply:
@sxyazi Solved :)